README.md

```diff
@@ -12,21 +12,52 @@ add and remove code as you see fit, for modding or research purposes.
 It builds the following DOL:
 v1.02 - main.dol: `sha1: 08e0bf20134dfcb260699671004527b2d6bb1a45`
 
-## Building
+# Building
 
-### Required tools
+## Required tools
 
 * [devkitPro](https://devkitpro.org/wiki/Getting_Started)
 * Python3 (`pacman -S msys/python3`)
 * gcc (`pacman -S gcc`)
 
-### Instructions
+## Instructions
 
 1. Download GC_WII_COMPILERS.zip from (https://cdn.discordapp.com/attachments/727918646525165659/917185027656286218/GC_WII_COMPILERS.zip) and extract the GC compilers to tools/mwcc_compiler/.
 2. Run the `make` command
 
+# Contributing
+
+If you're new to decomp or getting started, check out our [Getting Started guide](GETTING_STARTED.md)! Once you're in a place that you're ready to open a PR, check out our [Contributing guidelines](CONTRIBUTING.md).
+
+# FAQ
+## Is this legal?
+
+- Yes. You require an ISO to build this and no assets are here.
```

Comment:
I don't think we should make legal claims like this, at least not without further citation.